### PR TITLE
Change nctl config with env variables

### PR DESCRIPTION
**Pull request by davidatwhiletrue**
* Resolves: #12

### Summary

Currently deploys take 1 minute to process. On the other hand, blocks are added to the blockchain at a speed that ranges from 4s to 8min. These times can be tweaked changing the iight parameters in `chainspec.toml` and `config.toml`.
The `restart` script in the container has been modified to use environment variables to change the default settings. It also includes a variable to enable the usage of predefined accounts, removing the separate script.

### TODO

~

### Checklist

- [x] Code is properly formatted
- [x] All commits are signed
- [x] Tests included/updated or not needed
- [x] Documentation has been updated or is not required

### CODE REVIEW

**Comment by davidatwhiletrue** [comment]
Image `casper-nctl:v144-220303` in Docker Hub includes these changes-

File: README.md [hunk 1]
```diff
@@ -48,12 +55,31 @@ nctl-view-user-secret-key user=3
...
 ```
 
+### Change the settings using environment variables
+
+Using environment variables you can tweak few parameters to change the frequency of blocks and the  deploys processing speed.
+
+| Environment variable   | Default value | Description                                                                                                                                                                                                                                                                           |
+|------------------------|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| MINIMUM_ROUND_EXPONENT | 12            | Integer between 0 and 255.  The power of two that is the number of milliseconds in the minimum round length, and therefore the minimum delay between a block and its child.  E.g. 12 means 2^12 milliseconds, i.e. about 4 seconds.                                                   |
```
Comment (mrkara) [inline README.md:70]: Are 0 and 255 included? If so, making it explicitly known might be useful.

File: README.md [hunk 2]
```diff
@@ -48,12 +55,31 @@ nctl-view-user-secret-key user=3
...
+| MAXIMUM_ROUND_EXPONENT | 19            | Integer between 0 and 255.  Must be greater than `minimum_round_exponent`.  The power of two that is the number of milliseconds in the maximum round length, and therefore the maximum delay between a block and its child.  E.g. 19 means 2^19 milliseconds, i.e. about 8.7 minutes. |
```
Comment (mrkara) [inline README.md:71]: Are 0 and 255 included? If so, making it explicitly known might be useful.

**Review by mrkara — CHANGES_REQUESTED** [review]
Looks great. Just a few nitpicks.

File: README.md (hunk 1 above)
Comment (davidatwhiletrue) [reply README.md]: Fixed. 0 is included, 255 is not.
Added also a warning note. Too low or too high values will lead to nodes malfunction even allowed.

File: README.md (hunk 2 above)
Comment (davidatwhiletrue) [reply README.md]: Fixed. 255 is included, 0 is not.
Added also a warning note. Too low or too high values will lead to nodes malfunction even allowed.

**Review by mrkara — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**